examples/flux-todomvc/js/actions/TodoActions.js

```diff
@@ -78,6 +83,7 @@ var TodoActions = {
     AppDispatcher.dispatch({
       actionType: TodoConstants.TODO_DESTROY_COMPLETED
     });
+    console.log(TodoConstants.TODO_DESTROY_COMPLETED);
```

Comment:
If we do these `console.log()` lines, people are going to think that this is a best practice.  I think we should let people put in their own `console.log()`.